master/internal/command/tensorboard_manager.go

```diff
@@ -96,11 +104,63 @@ func (t *tensorboardManager) Receive(ctx *actor.Context) error {
 		}
 
 		actors.NotifyAfter(ctx, tickInterval, tensorboardTick{})
+	case TensorboardRequestWithUser:
+		summaryResp, err := t.handleTensorboardRequest(ctx, msg.User, &msg.Tensorboard)
+		if err != nil {
+			return errors.Wrap(err, "failed to launch tensorboard")
```

Comment:
I think we want to respond with an error, but do we want to return it here? This will crash the tensorboard manager (notice the errors weren't returned in the old code, just `ctx.Respond()`'ed).